README.md

```diff
@@ -44,6 +46,10 @@ If a potential supplier submits a bid for multiple lots as a single document, fo
 
 If the bid cannot be divided (for example, the data source describes only the total value of the bid, and not the individual value for each lot within the bid), create one `Bid` object, and add all lots' identifiers to the object's `relatedLots`.
 
+### Publish expressions of interest
+
+Buyers and procuring entities can publish expressions of interest, which are received during pre-qualification or pre-selection, in the `bids.details` array. Use the `bids.details.submissionType` field to indicate if the published data refer to a bid or an expression of interest.
```

Comment:
Buyers do not do the publishing. If we want to describe expressions of interest, that should be in the intro or in the schema sections of the readme.

```suggestion
Expressions of interest are also disclosed in the `bids.details` array. Use the `bids.details.submissionType` field to indicate whether a submission is a bid or an expression of interest.
```